src/why3prover.ml

```diff
@@ -0,0 +1,79 @@
+(** Calling provers in Why3 *)
+
+open Timed
+
+(** [!current_prover] contains the name of the current prover (can be changed
+    by using the set prover <name> command). *)
+let current_prover : string ref = ref "Alt-Ergo"
```

Comment:
Please, rename it into default_prover (as it can be overridden locally).

Reply:
Renamed in a29414085332857be96b62ab67a96960b46d746d